<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_gestaoesotque</class>
 <widget class="QDialog" name="frm_gestaoesotque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestão de Estoque</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>761</width>
     <height>630</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Novo Produto</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_9">
     <property name="geometry">
      <rect>
       <x>543</x>
       <y>566</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="btn_nv_novo">
        <property name="text">
         <string>Novo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_nv_salvar">
        <property name="text">
         <string>Salvar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>344</y>
       <width>741</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>Descrição Peça:</string>
     </property>
     <widget class="QWidget" name="layoutWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>92</y>
        <width>171</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Nc. Peça</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nv_ncPeca"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>149</y>
        <width>171</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Denominação</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nv_denominacao"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_6">
      <property name="geometry">
       <rect>
        <x>196</x>
        <y>30</y>
        <width>171</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Grupo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nv_grupo"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>197</x>
        <y>92</y>
        <width>120</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Tipo de Peça</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nv_tipoPeca"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_10">
      <property name="geometry">
       <rect>
        <x>394</x>
        <y>151</y>
        <width>101</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Pos. Prateleira</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_nv_posicao_prateleira"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_7">
      <property name="geometry">
       <rect>
        <x>393</x>
        <y>92</y>
        <width>61</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Qtde</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nv_qtde"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>392</x>
        <y>30</y>
        <width>121</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Valor de Venda</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nv_valorvenda"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>197</x>
        <y>149</y>
        <width>121</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Valor de Compra</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nv_valorcompra"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>131</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_19">
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Código Fonecedor:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nv_codigoFornecedor">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QTableWidget" name="tw_nv_listafornecedores">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>71</y>
       <width>741</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>141</width>
       <height>50</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Filtrar por</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_nv_filtrar"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_17">
     <property name="geometry">
      <rect>
       <x>416</x>
       <y>38</y>
       <width>331</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="txt_nv_filtrar"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_nv_filtrar">
        <property name="text">
         <string>Filtrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>groupBox</zorder>
    <zorder>layoutWidget_9</zorder>
    <zorder>tw_nv_listafornecedores</zorder>
    <zorder>layoutWidget_16</zorder>
    <zorder>layoutWidget_17</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Gestão Produtos</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>572</x>
       <y>566</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_ge_salvar">
        <property name="text">
         <string>Salvar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ge_excluir">
        <property name="text">
         <string>Excluir</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTableWidget" name="tw_ge_listapecas">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>66</y>
       <width>741</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>50</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Filtrar por</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_ge_filtrar"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_19">
     <property name="geometry">
      <rect>
       <x>416</x>
       <y>33</y>
       <width>331</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="txt_ge_filtrar"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ge_filtrar">
        <property name="text">
         <string>Filtrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>339</y>
       <width>741</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>Descrição Peça:</string>
     </property>
     <widget class="QWidget" name="layoutWidget_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>171</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Nc. Peça</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_ge_ncPeca"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>171</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Denominação</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_ge_denominacao"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>171</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Grupo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_ge_grupo"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_14">
      <property name="geometry">
       <rect>
        <x>197</x>
        <y>30</y>
        <width>120</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Tipo de Peça</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_ge_tipoPeca"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_15">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>90</y>
        <width>101</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Pos. Prateleira</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_ge_posicao_prateleira"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_20">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>61</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_16">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Qtde</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_ge_qtde"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_21">
      <property name="geometry">
       <rect>
        <x>197</x>
        <y>150</y>
        <width>121</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_17">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Valor de Venda</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_ge_valorvenda"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_22">
      <property name="geometry">
       <rect>
        <x>197</x>
        <y>90</y>
        <width>121</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_18">
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Valor de Compra</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_ge_valorcompra"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
